docs/sdk/client-side-sdks/javascript.md

```diff
@@ -5,16 +5,18 @@ sidebar_position: 2
 
 # DevCycle JavaScript Client SDK
 
-The DevCycle JS SDK, like all client SDKs, will retrieve a configuration for the provided User upon initialization. The configuration which is retrieved contains all of the Features and Variables, meaning no further outbound network calls will be made to retrieve Feature information unless explicitly specified. 
+The JS SDK uses local bucketing to perform all user segmentation. Furthermore, local bucketing occurs within the SDK, providing users with immediate responses to variable and feature requests.
 
 The JS SDK is available on NPM as an open-source package that can be viewed on the DevCycle GitHub.
 
+When initialized, the SDK will download the latest version of your DevCycle environments’ configuration from a CDN.
```

Comment:
When initialized it downloads a config specific to the user provided and for the environment specific to the sdk key provided